### DISCUSSION TITLE
Ritradotto intro e altro inerente

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#49] [Generale]
Nella cartella "downtown la", trovate il file "prince1.dlg", che va in Unofficial patch\dlg\downtown la.
vi ho solo inserito la traduzione dell'intro, il resto ancora non lo tocco.

Nella cartella "haven", trovate il file "female_sire_whisper2.lip - (corretta la frase, da "vorrei", in "voglio".
che va in Unofficial patch\sound\cinematic\santa monica\haven.

Nella cartella "nines", trovate i file "line907_col_e.lip - "line908_col_e.lip - "line909_col_e.lip". (tradotte per l'intro, non lo erano).
che vanno in Unofficial patch\sound\character\dlg\main charaters\nines.

Nella cartella "prince1", trovate tutti i file lip che servono per i sottotitoli dell'intro. (ritradotto).

L'intro adesso è completa.
[Ritradotto intro e altro.zip](https://github.com/tesivo/VTMB_italian_translation_for_Unofficial_Patch/files/7857450/Ritradotto.intro.e.altro.zip)

**Comment by NarakuITA** [comment]
Ottimo lavoro!

**Reply by crazyLillo** [reply]
Grazie mille!

ma mi sono accorto che su una frase c'è qualcosina che non va.
non me ne sono accorto subito perchè la frase si legge benissimo.
ma è il nome di chi la dice che non si vede.
al posto di "Principe", leggo "Qualcuno" .
devo aver sbagliato il numero dopo char forse.
poi controllo,
anche su tutto quello che sto traducendo, potrei sempre revisionare i testi perchè magari mi viene una frase o termini migliori.
diciamo che sto alla versione 1.0...:-)

**Reply by crazyLillo** [reply]
ecco il file corretto, avevo solo scambiato le cifre decina unità dopo char.
[file corretto.zip](https://github.com/tesivo/VTMB_italian_translation_for_Unofficial_Patch/files/7863024/file.corretto.zip)

**Reply by tesivo** [reply]
Perfetto!
